src/pages/index.tsx

```diff
@@ -9,7 +10,7 @@ function Homepage(): ReactElement {
   if (__config.flags.enable_new_homepage_view) {
     return (
       <>
-        <p>New homepage</p>
```

Comment:
One day in the future when GitHub copilot is super magical maybe this one line is all we'll need 🤣